src/test/blackbox.rs

```diff
@@ -5,36 +5,45 @@ use crate::{
 	types,
 };
 use jsonrpc_core_client::{transports::local, RpcChannel};
+use crate::node::TestRuntimeRequirements;
 
-pub enum BlackBoxNode<Runtime> {
+/// A black box node, either runs a manual seal background node,
```

Comment:
```suggestion
/// A black box node, either runs a background node,
```
It shouldn't be `ManualSeal`, no? In `BlackBox` we assume the blocks are created, without our control.